source/fundamentals/crud/read/limit.txt

```diff
@@ -0,0 +1,119 @@
+first and the limit only applies to the documents left over after
+the skip.
+
+The examples below demonstrate, respectively, how to insert data into
+a collection, how to use ``limit()`` to restrict the number of returned documents,
+and how to combine ``limit()`` with ``skip()`` to further narrow the results returned from a query. 
+
+The following operation inserts documents representing books into a collection:
+
+.. code-block:: java
+
+    collection.insertMany(Arrays.asList(
+        new Document().append("_id", 1)
+            .append("title", "The Brothers Karamazov").append("length", 824)
+            .append("author", "Dostoyevsky"),
+        new Document().append("_id", 2)
+            .append("title", "Les Misérables").append("length", 1462).append("author", "Hugo"),
+        new Document().append("_id", 3)
+            .append("title", "Atlas Shrugged").append("length", 1088).append("author", "Rand"),
+        new Document().append("_id", 4)
+            .append("title", "Infinite Jest").append("length", 1104).append("author", "Wallace"),
+        new Document().append("_id", 5)
+            .append("title", "Cryptonomicon").append("length", 918).append("author", "Stephenson"),
+        new Document().append("_id", 6)
+            .append("title", "A Dance with Dragons").append("length", 1104)
+            .append("author", "Martin")
+    ));
+
+The next example queries the collection to return the top three
+longest books. It first matches all the documents with the query, then sorts on the
+``length`` field to return books with longer lengths before
+books with shorter lengths. Lastly, it limits the return value to ``3`` documents:
+
+.. code-block:: java
+  :emphasize-lines: 6
+
+    import com.mongodb.client.model.Sorts;
+
+    // define a cursor that will return the first 3 sorted items
+    MongoCursor<Document> cursor = collection.find()
+        .sort(descending("length"))
+        .limit(3)
+        .iterator();
+    // print out items
+    try {
+        while (cursor.hasNext()) {
+            System.out.println(cursor.next());
+        }
+    } 
+    // close the cursor
+    finally {
+        cursor.close();
+    }
+
+
+The code example above prints out the following three documents, sorted by
+length:
+
+.. code-block:: java
+
+    Document{{_id=2, title=Les Misérables, author=Hugo, length=1462}}
+    Document{{_id=6, title=A Dance with Dragons, author=Martin, length=1104}}
+    Document{{_id=4, title=Infinite Jest, author=Wallace, length=1104}}
+
+.. note::
+
+   The order in which you call ``limit()`` and ``sort()`` does not matter
+   because the driver reorders the calls to apply the sort first and the
+   limit after it. The following two calls are equivalent:
+
+   .. code-block:: java
+
+      collection.find().sort(descending("length")).limit(3);
+      collection.find().limit(3).sort(descending("length"));
+
+
+To see the next three longest books, append the ``skip()`` method to your 
```

Comment:
Given the recent activity on DOCS-9994  and DOCSP-13487 , I'm not sure we should be recommending this sort of usage of skip/limit.  Do we do this in other places?